libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -317,6 +317,20 @@ func writeFile(dir, file, data string) error {
 	return nil
 }
 
+func writeFileIfExist(dir, file, data string) error {
+	if dir == "" {
```

Comment:
Why dir is empty then write file.

Reply:
`writeFile` will return error if dir is empty.
Although it is not so good, let us just leave them to keep compatibility.